Vic2ToHoI4/Source/V2World/Relations.cpp

```diff
@@ -53,6 +53,10 @@ Vic2::Relations::Relations(const std::string& theTag, std::istream& theStream):
 		commonItems::singleString dateString(theStream);
 		truceUntil = date(dateString.getString());
 	});
+	registerKeyword(std::regex("influence_value"), [this](const std::string& unused, std::istream& theStream){
```

Comment:
You can now do:
```registerKeyword("influence_value", [this](const std::string& unused, std::istream& theStream){```

for better performance

Reply:
Check